packages/core/contracts/external/avm/interfaces/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+Any modifications to original source code can be found in this document. Original sources are also listed here.
```

Comment:
? this is new

Reply:
We've had `CHANGELOG` for other sub directories in `external`